packages/next/test/fixtures/00-app-dir/package.json

```diff
@@ -3,5 +3,8 @@
     "next": "canary",
     "react": "experimental",
     "react-dom": "experimental"
+  },
+  "scripts": {
+    "vercel-build": "next build"
```

Comment:
```suggestion
    "build": "next build"
```

Lets use the canonical `build`

Reply:
I have no idea why that file is changed lol